src/firefox/index.js

```diff
@@ -249,9 +250,51 @@ export async function useProfile(
     app,
     configureThisProfile = configureProfile,
     customPrefs = {},
+    searchProfilesPath = '',
   }: UseProfileParams = {},
 ): Promise<FirefoxProfile> {
-  const profile = new FirefoxProfile({destinationDirectory: profilePath});
+  let profile;
+  const finder = new FirefoxProfile.Finder(searchProfilesPath);
+  const finderGetPath = promisify(finder.getPath, finder);
+  const hasProfileName = (profileName) => {
+    return finder.profiles.filter(
+      (profileDef) => profileDef.Name === profileName
+    ).length !== 0;
+  };
+
+  try {
+    const dirExists = await isDirectory(profilePath);
+    const defaultProfilePath = (
+      hasProfileName('default') && await finderGetPath('default')
+    );
+    const defaultDevProfilePath = (
+      hasProfileName('dev-edition-default') &&
+        await finderGetPath('dev-edition-default')
+    );
+    if (dirExists) {
+      log.debug(`Copying profile directory from "${profilePath}"`);
```

Comment:
This message does not seem right here, `useProfile` is never going to copy a profile.